ui/src/widgets/ui-number-input/UINumberInput.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+    <v-tooltip :text="tooltip" :disabled="!tooltip?.length" location="bottom">
+        <!-- eslint-disable-next-line vue/no-template-shadow -->
+        <template #activator="{ props }">
+            <v-number-input
+                v-model="value" :reverse="false" controlVariant="default" :hideInput="false" :inset="false"
+                v-bind="props" :disabled="!state.enabled" class="nrdb-ui-number-field" :label="label"
+                :rules="validation" :clearable="clearable" variant="outlined" hide-details="auto"
+                :prepend-icon="prependIcon" :append-icon="appendIcon" :append-inner-icon="appendInnerIcon"
+                :prepend-inner-icon="prependInnerIcon" @update:model-value="onChange" @click:clear="onClear"
+            />
+        </template>
+    </v-tooltip>
+</template>
+
+<script>
+// eslint-disable-next-line import/no-unresolved
+import { VNumberInput } from 'vuetify/labs/VNumberInput'
+import { mapState } from 'vuex' // eslint-disable-line import/order
+
+export default {
+    name: 'DBUITextInput',
+    components: {
+        VNumberInput
+    },
+    inject: ['$socket', '$dataTracker'],
+    props: {
+        id: { type: String, required: true },
+        props: { type: Object, default: () => ({}) },
+        state: { type: Object, default: () => ({}) }
+    },
+    data () {
+        return {
+            delayTimer: null,
+            textValue: null
+        }
+    },
+    computed: {
+        ...mapState('data', ['messages']),
+        label () {
+            return this.props.label
```

Comment:
Let's make the `label` dynamic too please?

Reply:
fixed. now the label and `clearable` it also supporting